src/Cache/Cache.php

```diff
@@ -155,6 +155,7 @@ public static function registry(ObjectRegistry $registry = null)
      * @param string $name Name of the config array that needs an engine instance built
      * @return void
      * @throws \InvalidArgumentException When a cache engine cannot be created.
+     * @throws \Exception
```

Comment:
I only see InvalidArgumentException here, the RuntimeException is caught and handled.